x-pack/plugins/serverless_search/public/application/components/overview.tsx

```diff
@@ -167,7 +165,7 @@ export const ElasticsearchOverview = () => {
         bottomBorder="extended"
         data-test-subj="cloud-details-section"
       >
-        <CloudDetailsPanel cloudId={cloud.cloudId} elasticsearchUrl={cloud.elasticsearchUrl} />
+        <CloudDetailsPanel cloudId={cloud.cloudId} elasticsearchUrl={elasticsearchUrl} />
```

Comment:
```suggestion
        <CloudDetailsPanel cloudId={cloud.cloudId} elasticsearchUrl={elasticsearchUrl || undefined} />
```

I don't think the `CloudDetailsPanel` will handle `elasticsearchUrl== ''` correctly, sending undefined to the prop will ensure the component defaults it to the place holder instead of taking `''` as the value.